tests/unit/components/DocumentationTopic/Hierarchy.spec.js

```diff
@@ -624,6 +624,9 @@ describe('Hierarchy', () => {
         parentTopicIdentifiers: [],
         isSymbolBeta: true,
       },
+      stubs: {
```

Comment:
What's the reason we have to manually add these stubs in lots of places now? Is the newer test utils just more strict about only auto-stubbing the first level of subcomponents or something like that? Or is there something more specific about the way we wrote these tests where this kind of change is needed?

Reply:
Because slots in Vue 2.7 have changed slightly, when using `v-slot` and `#foo` internally. When components are stubbed, named slots or scoped slots do not render at all, so I had to pass the real components as stubs, in order to get tests to start passing.